packages/api-explorer/src/components/AuthForm/index.jsx

```diff
@@ -35,37 +38,30 @@ function getSecuritySections(securityTypes, config, onChange, onSubmit, schemeNa
   const {authInputRef, oauth, auth} = config
   return (
     <form 
-      onSubmit={(e) => {
+      onSubmit={e => {
         e.preventDefault() 
         onSubmit()
       }}
     >
-      <div>
-        {
-          Object.keys(securityTypes).map((type, index) => {
-            const securities = securityTypes[type];
-            return (
-              <div key={`security-${type + index}`} >
-                <div style={{padding: '15px 17px'}}>
-                  <section>
-                    {
-                      securities.map(security => (
-                        <SecurityInput
-                          {...{ auth, onChange, authInputRef, oauth }}
-                          key={security._key}
-                          scheme={security}
-                          schemeName={schemeName}
-                        />
-                      ))
-                    }
-                  </section>
+      {
+        Object.entries(securityTypes).map(([type, securities], index) => (
+          <section key={`security-${type + index}`}>
```

Comment:
can you extract the map in an indipendent function?

Reply:
I tried a few refactorizations but since a lot of parameters are deep into the function, the advantages are relative.